src/GraphQl/Resolver/Factory/CollectionResolverFactory.php

```diff
@@ -107,7 +107,7 @@ public function __invoke(string $resourceClass = null, string $rootClass = null,
                 $offset = 1 + (int) $after;
             }
 
-            $data = ['edges' => [], 'pageInfo' => ['endCursor' => null, 'hasNextPage' => false]];
+            $data = ['totalCount' => 0.0, 'edges' => [], 'pageInfo' => ['endCursor' => null, 'hasNextPage' => false]];
```

Comment:
Why do you use a float ?

Reply:
Because every number in our pagination is a float ?

Reply:
Indeed it's ok like this. It allows to deal with large(r) numbers, and in JSON it will be a `Number` anyway.

Reply:
@jewome62 @soyuka Because, when the collection is not empty, the `getTotalItems` method called further returns a float.